Documentation/Books/Manual/Administration/Configuration/GeneralArangod.md

```diff
@@ -28,6 +28,19 @@ Whether or not this option is specified, the server will always perform a
 version check on startup. Running the server with a non-matching version number
 in the VERSION file will make the server refuse to start.
 
+### Storage Engine
+As of ArangoDB 3.2 several storage engines are supported. The previously existing
+engine is called `MMFiles`, the newly created one is based on [RocksDB](http://rocksdb.org)
```

Comment:
engine is called `MMFiles`, which is also the default storage engine. An alternative engine based on [RocksDB](http://rocksdb.org) is also provided and can be turned on manually.